.github/workflows/super-linter.yml

```diff
@@ -0,0 +1,18 @@
+name: Super-Linter
```

Comment:
(not sure if you want feedback while you're iterating — let me know either way! — if you do — this is reasonable but let's add to `pull-request.yaml`; it doesn't need its own workflow. And fine to PR things in separately if that lets you move faster)